json-spec/examples/landsat8-sample.json

```diff
@@ -35,92 +35,195 @@
  
   "properties": {
     "datetime": "2014-06-02T09:22:02Z",
-    "provider": "LANDSAT_8",
+    "provider": "USGS",
     "license": "PDDL-1.0",
-
-    "eo:off_nadir_angle" : -0.001,
+    "eo:platform": "landsat-8",
+    "eo:instrument": "OLI_TIRS",
+    "eo:collection": "L1T",
     "eo:cloud_cover" : 10.31,
+    "eo:off_nadir" : -0.001,
+    "eo:azimuth": 0.00,
     "eo:sun_azimuth" : 149.01607154,
     "eo:sun_elevation" : 59.21424700,
-    "eo:resolution" : 30,
-
-    "ls:data_type": "L1T",
-    "ls:wrs_path": 153,
-    "ls:wrs_row": 25,
-    "ls:earth_sun_distance": 1.0141560,
-    "ls:ground_control_points_verify": 114,
-    "ls:geometric_rmse_model": 7.562,
-    "ls:image_quality_tirs": 9,
-    "ls:ground_control_points_model": 313,
-    "ls:geometric_rmse_model_x": 5.96,
-    "ls:geometric_rmse_model_y": 4.654,
-    "ls:geometric_rmse_verify": 5.364,
-    "ls:image_quality_oli": 9
-
+    "landsat:wrs_path": 153,
+    "landsat:wrs_row": 25,
+    "landsat:earth_sun_distance": 1.0141560,
+    "landsat:ground_control_points_verify": 114,
+    "landsat:geometric_rmse_model": 7.562,
+    "landsat:image_quality_tirs": 9,
+    "landsat:ground_control_points_model": 313,
+    "landsat:geometric_rmse_model_x": 5.96,
+    "landsat:geometric_rmse_model_y": 4.654,
+    "landsat:geometric_rmse_verify": 5.364,
+    "landsat:image_quality_oli": 9
   },
 
   "links": [
     { "rel":"self", "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00.json"},
     { "rel":"alternate", "href": "https://landsatonaws.com/L8/153/025/LC81530252014153LGN00", "type": "html"},
-    { "rel":"source_metadata", "href":  "LC81530252014153LGN00_MTL.txt"},
     { "rel":"catalog", "href":  "http://landsat-pds.s3.amazonaws.com/L8/catalog.json"}
   ],
 
-  
   "assets" :{
     "thumbnail": {
-          "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_thumb_large.jpg"
+          "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_thumb_large.jpg",
+          "required": true,
+          "type": "jpeg"
+    },
+    "metadata": {
+      "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_MTL.txt",
+      "required": true,
+      "type": "mtl"
     },
     "B1": {
-      "href": "LC81530252014153LGN00_B1.TIF",
-      "product" : "http://landsat-pds.s3.amazonaws.com/L8/landsat8_product-band-1.json"
+      "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B1.TIF",
```

Comment:
Could that reference be s3://... ? AWS datasets that sits in Requester Payer buckets will not allow http:// requests without AWS authentication.

Reply:
Yes, s3 would be fine, if that's what the catalog represents. Sentinel l1c is moving to a requestor pays bucket this summer so any catalog for that will have to switch to using s3 addresses.

Reply:
Hrm, that is a good point on the requestor pays. I was contemplating going further on saying everything should be http:, but that's a good counterpoint.